package.json

```diff
@@ -47,8 +47,8 @@
     "sync-request": "^6.0.0",
     "universal-analytics": "^0.4.16",
     "uuid": "^8.3.0",
-    "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12"
+    "webpack": "^5.50.0",
```

Comment:
Going from Webpack 4 to 5 had some config changes that broke when I did it for Sirius Supervision. Presumably the build works here?

Reply:
I made sure that the build worked and tests passed, with a few (not many) manual tests in the app